src/test/java/com/appland/appmap/output/v1/CodeObjectTest.java

```diff
@@ -0,0 +1,31 @@
+package com.appland.appmap.output.v1;
+
+import javassist.ClassPool;
+import javassist.CtClass;
+import javassist.NotFoundException;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class CodeObjectTest {
+
+    @Test
+    public void getSourceFilePath_for_RegularClass() throws NotFoundException {
+        CtClass testCtClass = ClassPool.getDefault().get("com.appland.appmap.ExampleClass");
+        assertEquals("com/appland/appmap/ExampleClass.java", CodeObject.getSourceFilePath(testCtClass));
+    }
```

Comment:
This seems unlikely, but is there any way of us capturing the source path relative to the project root? If we wanted to open this file, we'd need to know if it was located in `src/java/test` or `src/java/main`. We could choose a file with the best partial match but without the full relative path we'll always have edge cases.

🤔